doc/source/contributing.rst

```diff
@@ -2,36 +2,136 @@
 Contributing
 ============
 
-pyLXD development is done `on Github <https://github.com/lxc/pylxd>`_. Pull
-Requests and Issues should be filed there. We try and respond to PRs and
-Issues within a few days.
+pyLXD development is done on `Github`_. Pull Requests and Issues should be
+filed there. We try and respond to PRs and Issues within a few days.
 
-If you would like to contribute large features or have big ideas, it's best
-to post on to `the lxc-users list
-<https://lists.linuxcontainers.org/listinfo/lxc-users>`_
-to discuss your ideas before submitting PRs.
+If you would like to contribute major features or have big ideas, it's best to
+post at the `Linux Containers disucssion forum
+<https://discuss.linuxcontainers.org/>`_ to discuss your ideas before
+submitting PRs.  If you use ``[pylxd]`` in the title, it'll make it clearer.
+
+Adding a Feature or Fixing a Bug
+--------------------------------
+
+The main steps are:
+
+1. There needs to be a bug filed on `Github`_ repository.  This is also for a
```

Comment:
"on the `Github` repository" ?

Reply:
Yup, that's where bugs are filed for the pyLXD.  The link is to this repository.

Reply:
I'm more complaining about the grammar: "There needs to be a bug filed on `Github`_ repository." is painful to read